src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+import { apiAddBook, apiGetBooks, apiRemoveBook } from '../../modules/api';
+
+const ADD = 'bookstore/Book/ADD';
+const GET = 'bookstore/Book/GET';
+const REMOVE = 'bookstore/Book/REMOVE';
+
+export const addBook = (payload) => async (dispatch) => {
```

Comment:
- [ ] Kindly ensure books added are stored in the API, currently, after page reload the books disappear.